op-monitorism/transaction_monitor/monitor.go

```diff
@@ -0,0 +1,161 @@
+package transaction_monitor
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/prometheus/client_golang/prometheus"
+    "github.com/ethereum-optimism/optimism/op-service/metrics"
+)
+
+const (
+	MetricsNamespace = "tx_mon"
+)
+
+type Monitor struct {
+	log           log.Logger
+	client        *ethclient.Client
+	watchConfigs  map[common.Address]WatchConfig // map of watch address to its config
+
+	// metrics
+	transactions         *prometheus.CounterVec
+	unauthorizedTx      *prometheus.CounterVec
+	thresholdExceededTx *prometheus.CounterVec
```

Comment:
I think we can probably do thresholds on the alerting side

Reply:
Instead of reporting unauthorized tx we can just report the total amount of ETH transferred